R/.gitignore

```diff
@@ -0,0 +1,3 @@
+GWAS_Tutorial_Manhattan.png
+globals.R
```

Comment:
Got it. I meant to prevent committing changes I made to path to the support folder on my computer.